.github/workflows/integration-test.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Setup operator environment
         run: |
           sudo snap install concierge --classic
-          sudo concierge prepare -p microk8s --microk8s-channel "1.27-strict/stable" --juju-channel "3.4/stable" --lxd-channel "5.20/stable" --extra-snaps astral-uv/latest/stable
+          sudo concierge prepare -p microk8s --microk8s-channel "1.31-strict/stable" --juju-channel "3.5/stable" --lxd-channel "5.21/stable" --extra-snaps astral-uv/latest/stable
```

Comment:
Another PR is already merged to use Juju 3.5/stable.

Reply:
Fixed the conflict